packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -1,3 +1,147 @@
+export interface ISemanticTextColors {
```

Comment:
@phkuo Just fyi here, I moved the text colors to a separate interface which ISemanticColors inherits. This allows the `<Text>` component to only allow pulling from semantic colors.

Reply:
@dzearing Can you keep ISemanticColors at the top of the file? It has the naming conventions and describes categories and such attached to it, and those should be in-your-face when you open this file.